apps/style/common.scss

```diff
@@ -1679,4 +1679,9 @@ a.WireframeButtons_active, div.WireframeButtons_active {
   }
 }
 
+#manageAssetsModal {
+  width: 800px;
+  margin-left: -400px;
+}
```

Comment:
Why did this style move to common.scss?

Reply:
This style was moved to common.scss from apps/style/applab/style.scss because #manageAssetsModal was what was making the sound library dialog in App Lab wide enough to fit all 3 columns. App Lab and Game Lab are both droplet labs, I thought they should have the same style so since common.scss is used by both App and Game, I moved #manageAssetsModal to common.scss instead of just applab.scss.

Reply:
:+1: Sounds good.